app/src/main/java/org/schabi/newpipe/settings/ContentSettingsManager.kt

```diff
@@ -17,8 +19,9 @@ class ContentSettingsManager(private val fileLocator: NewPipeFileLocator) {
      * It also creates the file.
      */
     @Throws(Exception::class)
-    fun exportDatabase(preferences: SharedPreferences, outputPath: String) {
-        ZipOutputStream(BufferedOutputStream(FileOutputStream(outputPath)))
+    fun exportDatabase(preferences: SharedPreferences, file: StoredFileHelper) {
+        file.create()
+        ZipOutputStream(BufferedOutputStream(SharpOutputStream(file.stream)))
```

Comment:
@wb9688 @Stypox Does that then mean, that the SharpOutputStream can be removed here?

Reply:
No, SharpOutputStream wraps file.stream (which is a SharpStream) so that it is possible to use it there.

Reply:
Ah now i understood it. thanks for clarifying